Source/Fuse.PushNotifications/Android/Impl.uno

```diff
@@ -318,7 +320,7 @@ namespace Fuse.PushNotifications
 				notificationBuilder.setSound(defaultSoundUri);
 			}
 
-			int id = PushNotificationReceiver.nextID();
+			int id = requestID;
```

Comment:
OK, reading this a bit more, I feel relatively sure I've misunderstand what the code tries to do in my explanation above. It would seem to me like `PushNotificationReceiver.nextID()` would be pretty much the perfect way of making each request unique. So why don't we want to use it?

Reply:
Okay, done i have changed it on my own local repo.